DEVELOPMENT.md

```diff
@@ -146,6 +146,12 @@ or any of the other Brokers available inside the `config/brokers/` directory.
 ko apply -f config/brokers/channel-broker/
 ```
 
+You may also need to install the MT Channel Based Broker
```

Comment:
Thank you for doing this. Since the MT Channel Based Broker is actually the only one now left in the repo, would you mind changing the above (L142) to just point to MT channel based broker instead of leaving that and adding this section?

Reply:
Done